wetmap/src/components/newModals/shopModal/index.tsx

```diff
@@ -14,14 +11,11 @@ import ShopModalView from './view';
 
 export default function ShopModal(props) {
   // const {lat, lng, setSelectedPhoto, setPhotoBoxModel } = props
-  // const { shopModal, setShopModal } = useContext(ShopModalContext);
   // const { selectedShop, setSelectedShop } = useContext(SelectedShopContext);
   // const [siteCloseState, setSiteCloseState] = useState(false);
```

Comment:
I assume these comment out lines that have not been deleted yet should be dealt with when merging into main with the new shopModal changes, correct?

Reply:
looks like these got taken care of my the back merge